tests/lax_scipy_special_functions_test.py

```diff
@@ -218,6 +218,12 @@ def testGammaSign(self):
     self._CheckAgainstNumpy(osp_special.gamma, lsp_special.gamma, args_maker, rtol=rtol)
     self._CompileAndCheck(lsp_special.gamma, args_maker, rtol=rtol)
 
+  def testScipySpecialFunNdtri(self):
+    dtype = jax.numpy.zeros(0).dtype
+    args_maker = lambda: [np.arange(-10, 10).astype(dtype)]
+    rtol = 1E-3 if jtu.test_device_matches(["tpu"]) else 1e-5
+    self._CheckAgainstNumpy(osp_special.ndtri, lsp_special.ndtri, args_maker, rtol=rtol)
+    self._CompileAndCheck(lsp_special.ndtri, args_maker, rtol=rtol)
```

Comment:
Rather than adding a new manual tests, perhaps we should update the bounds in the main test for more test coverage? https://github.com/google/jax/blob/00489be23d37d2ae0cdb9d9d0d98807ee8ae1ad6/tests/lax_scipy_special_functions_test.py#L115-L118

Reply:
Sure, sounds good. In that case I'd advocate for a more informative test name, like `testNdtriExtremeValues`

Reply:
Updated the bounds `low=0.0` and `high=1.0` for `ndtri` in `JAX_SPECIAL_FUNCTION_RECORDS`. Also, changed the name of the test at extreme values to `testNdtriExtremeValues` as suggested.